### PR TITLE
fix a typo in kubeadm v1beta4 doc

### PR DESCRIPTION
**Pull request by pacoxu**
#### What type of PR is this?
/kind documentation
/kind cleanup

#### What this PR does / why we need it:

 https://kubernetes.io/docs/reference/config-api/kubeadm-config.v1beta3/ is generated from https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1beta3/doc.go#L22

v1beta4 is similar

#### Which issue(s) this PR fixes:

Found when writing https://github.com/kubernetes/website/pull/47248
Fixes None

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

File: cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go [hunk 1]
```diff
@@ -27,21 +27,21 @@ limitations under the License.
 //   - Support custom environment variables in control plane components under `ClusterConfiguration`.
 //     Use `APIServer.ExtraEnvs`, `ControllerManager.ExtraEnvs`, `Scheduler.ExtraEnvs`, `Etcd.Local.ExtraEnvs`.
 //   - The ResetConfiguration API type is now supported in v1beta4. Users are able to reset a node by passing
-//     a --config file to "kubeadm reset".
-//   - `dry-run` mode in is now configurable in InitConfiguration and JoinConfiguration.
+//     a `--config` file to "kubeadm reset".
+//   - dry run mode is now configurable in InitConfiguration and JoinConfiguration.
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go:31]:
```suggestion
//   - Dry run mode is now configurable in InitConfiguration and JoinConfiguration.
```

File: cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go [hunk 2]
```diff
@@ -27,21 +27,21 @@ limitations under the License.
 //   - Support custom environment variables in control plane components under `ClusterConfiguration`.
 //     Use `APIServer.ExtraEnvs`, `ControllerManager.ExtraEnvs`, `Scheduler.ExtraEnvs`, `Etcd.Local.ExtraEnvs`.
 //   - The ResetConfiguration API type is now supported in v1beta4. Users are able to reset a node by passing
-//     a --config file to "kubeadm reset".
-//   - `dry-run` mode in is now configurable in InitConfiguration and JoinConfiguration.
+//     a `--config` file to "kubeadm reset".
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go:30]: should we make all `kubeadm ...` commands not use the `'` or `"` quotes and instead use the ` (back quote)

**Review by neolit123 — COMMENTED** [review]
i think we can merge this change in 1.31 during code freeze because it's just a docs bug fix (no risk).

/milestone v1.31

release team if you disagree, please let us know.

File: cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go (hunk 2 above)
Comment (pacoxu) [reply cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go]: Updated.

File: cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go [hunk 3]
```diff
@@ -27,21 +27,21 @@ limitations under the License.
...
 //   - Add `ClusterConfiguration.EncryptionAlgorithm` that can be used to set the asymmetric encryption algorithm
 //     used for this cluster's keys and certificates. Can be one of "RSA-2048" (default), "RSA-3072", "RSA-4096" or "ECDSA-P256".
 //   - Add `ClusterConfiguration.DNS.Disabled` and `ClusterConfiguration.Proxy.Disabled` that can be used to disable
 //     the CoreDNS and kube-proxy addons during cluster initialization. Skipping the related addons phases,
 //     during cluster creation will set the same fields to `false`.
-//   - Add the `NodeRegistration.ImagePullSerial` field in 'InitConfiguration` and `JoinConfiguration`, which
+//   - Add the `NodeRegistration.ImagePullSerial` field in `InitConfiguration` and `JoinConfiguration`, which
 //     can be used to control if kubeadm pulls images serially or in parallel.
-//   - The UpgradeConfiguration kubeadm API is now supported in v1beta4 when passing --config to "kubeadm upgrade" subcommands.
+//   - The UpgradeConfiguration kubeadm API is now supported in v1beta4 when passing `--config` to `kubeadm upgrade“ subcommands.
```
Comment (neolit123) [inline cmd/kubeadm/app/apis/kubeadm/v1beta4/doc.go:42]:
```suggestion
//   - The UpgradeConfiguration kubeadm API is now supported in v1beta4 when passing `--config` to `kubeadm upgrade` subcommands.
```

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Pull request merged**